README.rst

```diff
@@ -27,15 +27,29 @@ And an Array can be converted to a numpy ndarray on demand::
     array([ 0.72151309,  0.54654914], dtype=float32)
 
 
-Get in touch!
--------------
+------------------------------------------
+Important Note : Future development status
+------------------------------------------
+Biggus was originally developed primarily to provide large data handling for
+`Iris <http://github.com/SciTools/iris>`_ .
+
+For most applications it has now been surpassed by
```

Comment:
> "most" ... still some activities better handled by biggus

Well, I said that mainly because we should admit that masked data handling is still much easier with Biggus !
I did consider specifically mentioning that, but I feel that the immediate future of it is still unclear, so no point making comments that may rapidly go out of date.